tiledb/sm/cpp_api/array.h

```diff
@@ -315,7 +315,11 @@ class Array {
    * @param own=true If false, disables underlying cleanup upon destruction.
    * @throws TileDBError if construction fails
    */
-  Array(const Context& ctx, tiledb_array_t* carray, bool own = true)
+  Array(
+      const Context& ctx,
+      tiledb_array_t* carray,
+      bool own = true,
+      bool dontclose = false)
```

Comment:
I think we need to stick to snake case everywhere (words in small caps, separated by underscores). Let's be consistent with format, notation, etc.

Reply:
underscore added to name